lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+attr_reader :dishes
+    def initialize
```

Comment:
Right now this class has only one menu fixed, you can try to make it more flexible so that more items can be added